notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    fh = fits.open(fits_url)
</pre><p>After this, please explain the file extensions (HDU, also spell out please) and what each HDU contains.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>